### PR TITLE
PR: Initial merge with dev

### PR DESCRIPTION
**Pull request by rudolphpienaar**
Dockerfile: merge with @danmcp changes
pfcon/pfcon.py:
* Add pfmisc to handle/abstract out debug reporting
** Changes from self.qprint to self.dp.qprint
* Updates to status handling
* Updates to creating summary reports

### CODE REVIEW

**Comment by rudolphpienaar** [comment]
Tested e2e with pfioh/pman -- ready to merge.

File: README.rst [hunk 1]
```diff
@@ -17,7 +17,7 @@ pfcon  v1.3.3
 Overview
 ********
 
-This repository provides ``pfcon`` -- a controlling service that speaks to remote ``pman`` and ``pfioh`` services.
+This repository provides ``pfcon`` -- a controlling service that speaks to remote ``pman`` and ``pfioh`` services. This is the bleeding edge branch.
```
Comment (danmcp) [inline README.rst]: What is this extra line about?  Did that only apply to the dev branch?

File: pfcon/pfcon.py [hunk 2]
```diff
@@ -1034,19 +1090,22 @@ def coordinate_process(self, *args, **kwargs):
...
         """
 
-        self.qprint("coordinate_process()", comms = 'status')
+        global Gd_internalvar
+
+        self.dp.qprint("coordinate_process()", comms = 'status')
         b_status                    = False
         d_request                   = {}
-        d_jobOperation              = {}
+        d_jobStatus              = {}
```
Comment (danmcp) [inline pfcon/pfcon.py]: Formatting is off.

File: README.rst (hunk 1 above)
Comment (rudolphpienaar) [reply README.rst]: Yes -- pretty much. I'll remove

File: pfcon/pfcon.py (hunk 2 above)
Comment (rudolphpienaar) [reply pfcon/pfcon.py]: Thanks!

**Pull request merged**